link-grammar: 5.8.0 -> 5.8.1

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://repology.org/metapackage/link-grammar/versions.


meta.description for link-grammar is: "A Grammar Checking library"

meta.homepage for link-grammar is: "https://www.abisource.com/projects/link-grammar/"

meta.changelog for link-grammar is: ""


###### Updates performed
- Version update

###### To inspect upstream changes





###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed

- 0 of 0 passed binary check by having a zero exit code.
- 0 of 0 passed binary check by having the new version present in output.
- found 5.8.1 with grep in /nix/store/hy6rnzb07v3vb4yjv8bsd5ln7glcbx8j-link-grammar-5.8.1-bin
- directory tree listing: https://gist.github.com/ca8d7bc52a49488d7cb424ee02499190

- du listing: https://gist.github.com/3bb381cbf2ba3793e14d583a65992bfa

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
4 total rebuild path(s)

1 package rebuild(s)

1 x86_64-linux rebuild(s)
1 i686-linux rebuild(s)
1 x86_64-darwin rebuild(s)
1 aarch64-linux rebuild(s)


First fifty rebuilds by attrpath
link-grammar
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/hy6rnzb07v3vb4yjv8bsd5ln7glcbx8j-link-grammar-5.8.1-bin \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A link-grammar https://github.com/r-ryantm/nixpkgs/archive/dc912f45d764c20370a186e6a008bf844620b734.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/hy6rnzb07v3vb4yjv8bsd5ln7glcbx8j-link-grammar-5.8.1-bin
ls -la /nix/store/hy6rnzb07v3vb4yjv8bsd5ln7glcbx8j-link-grammar-5.8.1-bin/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>link-grammar</li>
  </ul>
</details>

---

###### Maintainer pings

cc @jtojnar for testing.

r-ryantm committed:
link-grammar: 5.8.0 -> 5.8.1

jtojnar reviewed (commented):
[Changelog](https://github.com/opencog/link-grammar/blob/link-grammar-5.8.1/ChangeLog) looks okay, [commits](https://github.com/opencog/link-grammar/compare/link-grammar-5.8.0...link-grammar-5.8.1) too.

jtojnar committed:
link-grammar: Improve

- Add meta.changelog
- Correct license
- Run tests
- Add quick smoke test to passthru.tests
